game/main.css

```diff
@@ -6,60 +6,74 @@ body {
     border-collapse: collapse;
     background: url(game_icons/board-game-bottom.png) no-repeat rgb(181, 228, 255);
     background-position: bottom;
-    background-position-y: 115%;
-    width: 820px;
-    height: 758px;
+    background-size: 100%;
```

Comment:
To nie do końca rozwiązuje nasz problem związany z wychodzeniem gry poza okno przeglądarki. Gra jest mniejsza i będzie w całości widoczna na mniejszych rozdzielczościach, ale na większych - np. full hd, które obecnie jest standardem, gra jest bardzo mała. Myślałem, że gra będzie się skalowała w zależności od rozdzielczości ekranu. Co ciekawe, gra się skaluje w dół dla ekranów o małych rozdzielczościach jak tablety i komórki.

Reply:
![cotek1](https://user-images.githubusercontent.com/37185264/54359519-19cbbd00-4663-11e9-80fd-725407912f75.png)
Dla zobrazowania wielkość okna gry na rozdzielczości full-hd

Reply:
Jeszcze jedna uwaga. Zmieniłaś wielkość pola gry, ale nie zmieniłaś rozdzielczości okna przeglądarki w pliku "game_for_mail.js", który odpala grę.

Reply:
Ja bym zostawil tak jak jest, czyli troche mniejsza niz byla. Jedynie by trzeba zmienic wielkosc okna tak, zeby pasowala do obecnego rozmiaru gry i tyle. Jakbysmy chcieli zrobic skalowanie to nie wiem jak ten problem obejsc, wiec tak sie wydaje latwiej.